rcode-reader/src/main/java/eu/rssw/pct/elements/VariableElement.java

```diff
@@ -0,0 +1,78 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.AccessibleClassElement;
+import eu.rssw.pct.DataType;
+import eu.rssw.pct.RCodeInfo;
+
+public class VariableElement extends AccessibleClassElement {
+  private static final int READ_ONLY = 1;
+  private static final int WRITE_ONLY = 2;
+  private static final int BASE_IS_DOTNET = 4;
+  private static final int NO_UNDO = 8;
+
+  private int data_type;
+  private int recordPosition;
+  private int extent;
+  private int flags;
+  private int fullNameLength;
+  private String typeName;
+
+  public VariableElement(String name, Set<AccessType> accessType, byte[] segment, int currentPos, int textAreaOffset, ByteOrder order) {
+    super(name, accessType);
+
+    this.data_type = ByteBuffer.wrap(segment, currentPos, Short.BYTES).order(order).getShort();
+    this.recordPosition = ByteBuffer.wrap(segment, currentPos + 2, Short.BYTES).order(order).getShort();
+    this.extent = ByteBuffer.wrap(segment, currentPos + 4, Short.BYTES).order(order).getShort();
+    this.flags = ByteBuffer.wrap(segment, currentPos + 6, Short.BYTES).order(order).getShort();
+    this.fullNameLength = ByteBuffer.wrap(segment, currentPos + 10, Short.BYTES).order(order).getShort();
+
+    int name_offset = ByteBuffer.wrap(segment, currentPos + 12, Integer.BYTES).order(order).getInt();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS00117)